openshift-kube-apiserver/admission/customresourcevalidation/node/validate_node_config.go

```diff
@@ -78,18 +84,21 @@ func validateConfigNodeForExtremeLatencyProfile(obj, oldObj *configv1.Node) *fie
 	return nil
 }
 
-func (configNodeV1) ValidateCreate(_ context.Context, uncastObj runtime.Object) field.ErrorList {
+func (c *configNodeV1) ValidateCreate(_ context.Context, uncastObj runtime.Object) field.ErrorList {
 	obj, allErrs := toConfigNodeV1(uncastObj)
 	if len(allErrs) > 0 {
 		return allErrs
 	}
 
 	allErrs = append(allErrs, validation.ValidateObjectMeta(&obj.ObjectMeta, false, customresourcevalidation.RequireNameCluster, field.NewPath("metadata"))...)
+	if err := nodelib.ValidateMinimumKubeletVersion(c.nodeLister, obj.Spec.MinimumKubeletVersion); err != nil {
```

Comment:
The field validation errors should probably be constructed here, instead of in library-go's ValidateMinimumKubeletVersion, because this is the place that actually encodes the path to the field in the API.